extensions/typescript-language-features/src/languageFeatures/completions.ts

```diff
@@ -500,7 +501,22 @@ class MyCompletionItem extends vscode.CompletionItem {
 		}
 	}
 
-	private static getCommitCharacters(context: CompletionContext, entry: Proto.CompletionEntry): string[] | undefined {
+	private static getCommitCharacters(
+		context: CompletionContext,
+		entry: Proto.CompletionEntry,
+		defaultCommitCharacters: readonly string[] | undefined): string[] | undefined {
+		// @ts-expect-error until TS 5.6
+		let commitCharacters = entry.commitCharacters ?? defaultCommitCharacters;
+		if (commitCharacters) {
+			if (context.enableCallCompletions
```

Comment:
Should this logic be moved to the TS side?

Reply:
In the future, yes. I didn't include that on TS Server for now because I'm still working out what our approach for commit characters will be exactly. It could be that this setting becomes irrelevant and we start offering `(` as a commit character on TSServer by default, or we could want more settings. But one way or the other, we'll eventually get rid of this logic on the vscode side.

Reply:
Ok that sounds good. I'll go ahead and merge this initial version then